action.yml

```diff
@@ -1,5 +1,5 @@
-name: 'API documentation on Bump'
-description: 'Generate your API doc and an automatic changelog from any OpenAPI or AsyncAPI file.'
+name: 'Bump.sh'
+description: 'Keep your OpenAPI and AsyncAPI, public, private and partner API docs synchronized with your code.'
```

Comment:
Reads strange to enumerate both OpenAPI and AsyncAPI together with “public”, “private” and “partner” doesn't it?

Reply:
I wanted to find a short sentence that explains Bump works for any type of API doc (public, private and partner) and any/multiple technologies (OpenAPI and AsyncAPI), while specifically naming it to appeal to readers.

Not sure how to twist this otherwise, would you have an idea?

Reply:
wdyt?
```suggestion
description: 'Keep your public, private and partner API documents in sync with your code, whether you use OpenAPI or AsyncAPI.'
```

Reply:
Reads nicer thanks both!